nifi-nar-bundles/nifi-azure-bundle/nifi-azure-processors/pom.xml

```diff
@@ -57,5 +81,10 @@ language governing permissions and limitations under the License. -->
             <version>${powermock.version}</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.apache.nifi</groupId>
+            <artifactId>nifi-standard-processors</artifactId>
```

Comment:
Please specify this dependency as provided, so that the processors in nifi-standard-processors aren't found in nifi-azure-nar as well as nifi-standard-nar.
```xml
        <dependency>
            <groupId>org.apache.nifi</groupId>
            <artifactId>nifi-standard-processors</artifactId>
            <version>${project.version}</version>
            <scope>provided</scope>
        </dependency>
```